### ISSUE TITLE
Leftmost bar wastes space

### ISSUE THREAD

**Issue by fbruetting** [vector-im/element-web#8733]
- **Browser**: Flatpak (so guess Chromium)
- **OS**: Fedora Silverblue 29

### Description

As you see in the screen shot below, the left-most bar wastes a lot of space. Additionally, I don’t think that the ruler makes any sense there. Can’t those two icons be put at the bottom of the contact/room list?

![bildschirmfoto von 2019-02-16 16-05-05](https://user-images.githubusercontent.com/10520767/52902489-9f4d8000-3211-11e9-82b9-74dbf515c6bb.png)

**Comment by Half-Shot** [comment]
![image](https://user-images.githubusercontent.com/2072976/52902572-af159600-320a-11e9-8dcf-c15f8e8cee18.png) 

It's meant to be for community icons, so at least for me it's not wasting any space.

**Comment by fbruetting** [comment]
Ah ok, then I have a proposal: These icons could be shown at the bottom of the conversation list like in the following image until they use the height of this space up to 20…30 % – then those icons could be outsourced to that separate pane.

![bildschirmfoto von 2019-02-16 18-16-09](https://user-images.githubusercontent.com/10520767/52902864-3bc65100-3217-11e9-89fb-245e47acb49d.png)

For getting people to use Riot/Matrix, I think it would be reasonable to cut the defaults down to the bare minimum „normal“ people would need/use. Riot then would look less complicated and hopefully get more people into using it. Advanced features are always cool, but should rather be hidden behind switches in the settings until users need or want it.

**Comment by nerwall** [comment]
Integrating the community icons into the same pane as contacts and rooms would significantly improve usability with a narrow window. 900px wide is plenty to use Signal, Gajim, Hexchat, and Mumble in, but riot is incredibly cramped.

**Comment by aladar42** [comment]
Is there no way to turn this bar off? I have no use for it, and it really does waste space for me.

**Comment by Half-Shot** [comment]
@aladar42 
![image](https://user-images.githubusercontent.com/2072976/53673465-9b911300-3c7f-11e9-9fd4-405747d2d570.png)

**Comment by aladar42** [comment]
That doesnt work on my client. Its off, but nothing changed, not even after a restart.

**Comment by aladar42** [comment]
Okay, I figured it out. Apparently I have to disable it, restart, then resize the room panel all the way to the max, then all the way to the min, at which point it disappears and I can resize it to the size I want. That... isnt the expected behavior, I would assume?

**Comment by Half-Shot** [comment]
I think that's https://github.com/vector-im/riot-web/issues/8418

**Comment by t3chguy** [comment]
Closing as that is indeed #8418

**Comment by fbruetting** [comment]
My issue is about the design. #8418 is about the settings not working.

**Comment by t3chguy** [comment]
I mean sure, but you can hide it if you don't like that, will re-open for discussion, maybe the setting could be a bit more automatic

**Comment by fbruetting** [comment]
If you don’t need it at all, yes. But as soon as you have one entry, there is a lot of wasted space, which I think should be improved, especially when the window is left/right tiled. ;)

**Comment by aaronraimist** [comment]
Related: #6494

**Comment by t3chguy** [comment]
The discussion boils down to #6494, closing in favour of it

**Comment by fbruetting** [comment]
#6494 is about totally hiding the panel when you’re not at all a member in a community. My issue in contrast is more about UI optimization – which especially is important if you’re in just a few communities. There, it would be disadvantageous if that panel would be completely hidden – but showing the current panel would also waste a lot of screen space.

That’s why I proposed to move the communities into the contacts panel if there are just a few. Ideal would be a responsive panel, which also shows the communities below or above the contacts (if they’re just a few) and then source them out into a separate area (i.e. current implementation) if the number of communities increases. Maybe also dependent on the width of this combined panel.

You’re sure you want to close this?

**Comment by t3chguy** [comment]
Will re-open to track a "combined left panel" mode sort of thing

**Comment by Caspy7** [comment]
Was asked to give my input here...

Apologies for any rehash, I don't currently have the brain juice to read through all the comments.
I complained about the wasted space on the left side. Fortunately I was in the right channel and someone told me how to hide it with a pref. 

I'd say this is a suboptimal situation as most users aren't going to readily discover this resolution. Heck, I scrolled through the prefs before and the had no idea what it was.

My idea on resolving this is rather than having an entire toolbar, place the panel as a box below the profile name/avatar (above the room list). Use the + like normal but when there are communities show them as overlapping, stacked circles. Expand them as a drawer on hover. If hover is too problematic a UX, then click to expand I suppose.
